test/functional/plugin/man_spec.lua

```diff
@@ -263,4 +263,14 @@ describe(':Man', function()
       { '1', 'other_man' },
     }, get_search_history('other_man(1)'))
   end)
+
+  it('can complete', function()
+    t.skip(t.is_os('mac') and t.is_arch('x86_64'), 'not supported on intel ma')
+    eq(
+      true,
+      exec_lua(function()
+        return #require('man').man_complete('g', 'Man g') > 0
```

Comment:
`:Man g` gets about 10x more results than `:Man f`. Just a guess, but maybe `:Man f` in this test will avoid potential timing-related issues.

Reply:
I can try, but this code is synchronous. I'm pretty sure the issue is the version of MacOS because one version updated all the coreutils and this is when `man -w` broke.

Reply:
Dam, I forgot to stage the change with the skip line commented out. Oh well.